src/Controller/TagCloudController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Searches;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+
+class TagCloudController extends AbstractController
+{
+    /**
+     * @Route("/tagcloud", name="tag_cloud")
+     */
+    public function index()
+    {
+        $data = $this->getData();
+        return $this->render('tag_cloud/index.html.twig', [
+            'controller_name' => 'TagCloudController', 'data' => $data
+        ]);
+    }
+    public function getData()
+    {
+        $entityManager = $this->getDoctrine()->getManager();
+        $searches = $entityManager->getRepository(Searches::class)->findTopHighestSearch();
+
+
+        // dd($search);
+
+        $data = [];
+
+        $maximum = 0;
+
+        foreach ($searches as $search) {
+
+            $counter = $search['searched'];
+
+            // dd($search);
+            $class = '';
+
+            if ($counter > $maximum) $maximum = $counter;
```

Comment:
always use { and } in if statements and remove not needed empty lines and comments